src/Microsoft.NetCore.Analyzers/UnitTests/Data/ReviewSQLQueriesForSecurityVulnerabilitiesTests_FlowAnalysis.cs

```diff
@@ -1,17 +1,35 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
 using Test.Utilities;
 using Xunit;
+using VerifyCS = Test.Utilities.CSharpSecurityCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Data.ReviewSqlQueriesForSecurityVulnerabilities,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
+using VerifyVB = Test.Utilities.VisualBasicSecurityCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Data.ReviewSqlQueriesForSecurityVulnerabilities,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
 
 namespace Microsoft.NetCore.Analyzers.Data.UnitTests
 {
     [Trait(Traits.DataflowAnalysis, Traits.Dataflow.ValueContentAnalysis)]
     public class ReviewSQLQueriesForSecurityVulnerabilitiesTests_FlowAnalysis : ReviewSQLQueriesForSecurityVulnerabilitiesTests
     {
+        private static DiagnosticResult GetCSharpResultAt(int line, int column, string invokedSymbol, string containingMethod)
```

Comment:
I am wondering if these should be helper methods which take line, column and a params string array so each test file doesn't have to define its own version. Not sure how often you had to add these to test files.

Reply:
I think @sharwell's idea is to get rid of all those helper methods and to use inline `VerifyCS.Diagnostic().WithLocation()...` but I am not sure.

To answer your question all analyzer tests have something similar, for some I have to specify the actual `DiagnosticDescriptor`, for some there is no arguments.

Reply:
Sounds good - you don't need to address this comment in this PR. We can discuss in the separate tracking issue if it is worth just having a single helper in VerifyCS/VerifyVB versus polluting every test file.

Reply:
https://github.com/dotnet/roslyn-analyzers/issues/3088